react/src/components/Chat/ChatHeader.tsx

```diff
@@ -199,7 +200,9 @@ const ChatHeader: React.FC<ChatHeaderProps> = ({
             }}
           />
         )}
-        <Button onClick={() => onClickCreate?.()} icon={<PlusOutlined />} />
+        {clonable && (
+          <Button onClick={() => onClickCreate?.()} icon={<PlusOutlined />} />
```

Comment:
Please add a tooltip such as "Add model for comparison", "Add chat", ...

Reply:
The other buttons on the chat page doesn't show any tooltip now. I don't feel that it's necessary ui.